landscape.yml

```
@@ -122,6 +122,13 @@ landscape:
             logo: joeynmt.svg
             repo_url: https://github.com/joeynmt
             crunchbase: https://www.crunchbase.com/organization/cornell-university
+          - item:
+            name: LitGPT
+            description: 20+ high-performance LLMs with recipes to pretrain, finetune and deploy at scale.
+            homepage_url: https://github.com/Lightning-AI/litgpt
+            logo: LitGPT Light Mode.svg
```

let's rather _ in the name to be consistent and prevent accidental breaking
```suggestion
            logo: LitGPT-light.svg
```